Automation_Grants.md

```diff
@@ -0,0 +1,5 @@
+Automation-License-grants
+
+Additional Use Grant(s):
+
+You may make use of Chainlink Automation (which is available subject to the license here the “Licensed Work ”) solely for purposes listed below:
```

Comment:
Just being super nit picky here but why is there a space after "Work" in `“Licensed Work ”`?

Reply:
dammit lol

Reply:
fixed